<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SandRBox</class>
 <widget class="QWidget" name="SandRBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>发送</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="te_sendData"/>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="cb_hexSend">
          <property name="text">
           <string>16进制发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Sclean">
          <property name="text">
           <string>清空显示</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_SbyHand">
          <property name="text">
           <string>手动发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="cb_rollSend">
          <property name="text">
           <string>循环发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>发送周期：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_periodic"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>毫秒</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>接收</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="cb_hexReceive">
          <property name="text">
           <string>16进制接收</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Rclean">
          <property name="text">
           <string>清空显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTextEdit" name="te_receivedData">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
